test/integration/install/test_install.py

```diff
@@ -0,0 +1,72 @@
+import re
+import subprocess
+import os
+
+
+def test_install_tt(tt_cmd, tmpdir):
+
+    configPath = os.path.join(tmpdir, "tarantool.yaml")
+    # Create test config with restart_on_failure true.
+    with open(configPath, 'w') as f:
+        f.write('tt:\n  app:\n    bin_dir:\n    inc_dir:\n')
+
+    # Install latest tt.
+    start_cmd = [tt_cmd, "install", "tt=master", "--cfg", configPath]
+    instance_process = subprocess.Popen(
+        start_cmd,
+        cwd=tmpdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+
+    # Check that the process was terminated correctly.
+    instance_process_rc = instance_process.wait()
+    assert instance_process_rc == 0
+
+    installed_cmd = [tmpdir + "/bin/tt", "run", "-v"]
+    installed_program_process = subprocess.Popen(
+        installed_cmd,
+        cwd=tmpdir + "/bin",
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+    start_output = installed_program_process.stdout.readline()
+    assert re.search(r"   • Running an instance...   \n", start_output)
+    run_output = installed_program_process.stdout.readline()
+    assert re.search(r"Tarantool", run_output)
```

Comment:
Could you explain where do tt find a tarantool? We installed only tt previously.

Reply:
I check that tt is ok, by running ./tt run -v which outputs version of Tarantool, changed to ./tt -h check